_posts/2018-09-03-rest-review.md

```diff
@@ -0,0 +1,99 @@
+---
+author: Jorge Gueorguiev Garcia
+layout: post
+asset-type: post
+title: "Rest Review"
+date: 2018-08-30 08:01:00
+description: Where I talk about what is, and what is not, the REST architecture.
+image: 
+    src: /assets/custom/img/blog/2018-08-13-automated-code-styling/caffeine-coffe-cup-6347.jpg
+tags: 
+- REST
+- Architecture
+---
+
+There are two events that have happened recently close to each other. The first one was a decision to actually learn GraphQl. I have been thinking about it for a while, but now is time to actually put the effort on it. The second one was a talk with my colleague [Luciano](https://codurance.com/publications/author/luciano-palma/) in which I incorrectly made a remark about resources on REST. Because of both (for the former as I want to compare, for the latter to assert my knowledge), I decided to re-read both the [Fielding PhD dissertation](https://www.ics.uci.edu/~fielding/pubs/dissertation/top.htm) and the book [Rest in Practice](http://restinpractice.com/book/) by Robinson, Webber and Parastatidis.
```

Comment:
Do you mean `GraphQL` here?

Reply:
When I mention resources on REST? No, that one is fine.

Reply:
Here: "The first one was a decision to actually learn GraphQl' -> "... GraphQL".